src/ontology/pato-edit.obo

```diff
@@ -20588,6 +20588,34 @@ def: "Existing or functioning as though broken into separate parts; disorganized
 is_a: PATO:0001444 ! broken
 created_by: http://orcid.org/0000-0001-5208-3432
 
+[Term]
+id: PATO:0040048
+name: absent physical object
+def: "A quality denoting the lack of an physical entity or object, such as a chemical." [thefreedictionary.:thefreedictionary.]
+subset: absent_slim
+synonym: "physical absence" EXACT []
+is_a: PATO:0001241 ! physical object quality
+created_by: http://orcid.org/0000-0001-5208-3432
+
+[Term]
+id: PATO:0040049
+name: absent process
+def: "A quality denoting the lack of an process or function." []
+subset: absent_slim
+synonym: "Process absence" EXACT []
+xref: thefreedictionary.:thefreedictionary.
+is_a: PATO:0001236 ! process quality
+created_by: http://orcid.org/0000-0001-5208-3432
+
+[Term]
+id: PATO:0040050
+name: absent anatomical entity
```

Comment:
Sounds a bit too much like it refers to a physical entity rather than a quality. Maybe 'absence of anatomical entity' ?

Reply:
I renamed this